schema/spacewalk/upgrade/susemanager-schema-4.3.6-to-susemanager-schema-4.3.7/100-add-suseMinionInfo-created-modified.sql

```diff
@@ -0,0 +1,23 @@
+ALTER TABLE suseMinionInfo ADD COLUMN IF NOT EXISTS
```

Comment:
The directory is wrong. 4.3.7 is already released. The next version is already 4.3.8.
So the directory should be named `susemanager-schema-4.3.7-to-susemanager-schema-4.3.8`

Reply:
Thanks @mcalmer. Should be fine now.